tests/playbooks/e2e.yaml

```diff
@@ -41,52 +41,18 @@
       retries: 18
 
     - name: Create a VM with the CDI PVC that was created
-      k8s:
+      kubevirt_vm:
        state: present
-       name: '{{ vm_name }}'
+       name: "{{ vm_name }}"
        namespace: default
-       inline:
-         apiVersion: kubevirt.io/v1alpha2
-         kind: VirtualMachine
-         metadata:
-           name: '{{ vm_name }}'
-           namespace: default
-           labels:
-             kubevirt.io/vm: test
-         spec:
-           running: false
-           template:
-             metadata:
-               labels:
-                 kubevirt-vm: cirros
-                 special: cirros-e2e-key
-             spec:
-               domain:
-                 resources:
-                   requests:
-                     memory: 512Mi
-                 cpu:
-                   cores: 1
-                 devices:
-                   disks:
-                   - volumeName: disk0-pvc
-                     name: mydisk
-                     disk:
-                       bus: virtio
-                   - name: cloudinitdisk
-                     volumeName: cloudinitvolume
-                     cdrom:
-                       bus: virtio
-               volumes:
-               - name: disk0-pvc
-                 persistentVolumeClaim:
-                   claimName: '{{ pvc_name }}'
-               - name: cloudinitvolume
-                 cloudInitNoCloud:
-                   userData: |
-                     #cloud-config
-                     password: "gocubsgo"
-                     chpasswd: ( expire: False )
+       memory: 64Mi
```

Comment:
You don't need the cloud-init here?

Reply:
the cirros image already has the cloudinit we're passing, so nothing specific about it
But I felt we should keep the `inline` object, so added it back now